termux-api.c

```diff
@@ -8,7 +8,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/endian.h>
+#ifdef __TERMUX
```

Comment:
Typo here, but apparently it works for termux as well if arpa/inet.h is included instead, so I'll update and include that header always